client/node.js

```diff
@@ -86,6 +87,10 @@ module.exports = client(function node (request) {
       reject(response)
     }
 
+    if (request.agent) {
```

Comment:
thanks for the tip and quick response!

that does make it even nicer, not having to alter `client/node.js`.

I'm working on a separate module to be used as a plugin, will update here once it's done on monday.